CHANGELOG.md

```diff
@@ -129,6 +129,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
     - Version checks for ProtocolLib and Shopkeepers support
     - `mmoitemtake` event and `mmoitem` condition - now also check the backpack
         - this will not work until the item rework / until the backpack contains NBT data
+- `first` event - attempts to run a list of events until one successfully runs, like a compressed `if` event
```

Comment:
This need to be moved above the line 123 `Things that are also added in 1.12.X:`